exporter/exporterhelper/traces_test.go

```diff
@@ -156,20 +146,30 @@ func TestTracesExporter_WithRecordEnqueueFailedMetrics(t *testing.T) {
 }
 
 func TestTracesExporter_WithSpan(t *testing.T) {
+	sr := new(oteltest.SpanRecorder)
+	tp := oteltest.NewTracerProvider(oteltest.WithSpanRecorder(sr))
+	otel.SetTracerProvider(tp)
+	defer otel.SetTracerProvider(trace.NewNoopTracerProvider())
```

Comment:
This is now the 5th time the same lines are repeated in tests. Wouldn't it make sense to externalize it?